Sources/SwiftProtobuf/Google_Protobuf_Timestamp_Extensions.swift

```diff
@@ -224,17 +207,30 @@ private func formatTimestamp(seconds: Int64, nanos: Int32) -> String? {
     // The following is crude, but it works.
     // TODO: If String(format:) works, that might be even better
     // (it was broken on Linux a while back...)
-    let result = (FormatInt(n: Int32(year), digits: 4)
+
+    func formatInt(n: Int32, digits: Int) -> String {
```

Comment:
Yeah I will definitely make another change to get rid of the custom string formatting as soon as I'm in a place to test it well enough.